src/svgs/CheckIcon.tsx

```diff
@@ -1,39 +1,16 @@
 import React from "react"
-import { color } from "../helpers"
-import { Color } from "../Theme"
 import { Icon, IconProps } from "./Icon"
 
-interface CheckIconProps extends IconProps {
-  fill?: Color
```

Comment:
I'm not sure of the best way to go about this, but interfaces containing things like `style`, `fill` sand so on need to be extended globally to `Icon.tsx` or individual icon interfaces need to be preserved. If `fill` is present  than that  means that this icon has collor applied  to it somewhere in one of our apps.